src/providers/seventv/eventapi/Subscription.hpp

```diff
@@ -70,10 +95,30 @@ constexpr magic_enum::customize::customize_t magic_enum::customize::enum_name<
     using chatterino::seventv::eventapi::SubscriptionType;
     switch (value)
     {
+        case SubscriptionType::AnyEmoteSet:
+            return "emote_set.*";
+        case SubscriptionType::CreateEmoteSet:
+            return "emote_set.create";
```

Comment:
Where are these used?

Reply:
These aren't used, they're only here for completeness. I can remove them if desired.

Reply:
Make it look like it does in the fork then for as few as possible merge conflicts